python/ray/_private/services.py

```diff
@@ -1258,6 +1254,11 @@ def read_log(filename, lines_to_read):
                             "Ray github. "
                             "https://github.com/ray-project/ray/issues"
                         )
+                logger.error(
+                    "Error should be written to 'dashboard.log' "
+                    "We are printing the last lines"
```

Comment:
can you keep the link to the log directory?

Reply:
```suggestion
                    "We are printing the last {len(lines)} lines"
```

Reply:
Both the log directory and the number of lines is contained a few lines lower. I think it should only appear once:
```
The last 2 lines of /tmp/ray/session_2023-05-16_10-04-16_493412_122666/logs/dashboard.log
(it may contain an error message from the dashboard): 

```